### PR TITLE
PMM-7649 Shows insufficient access permissions information

### PR DESCRIPTION
**Pull request by tiagomotasantos**
Some Percona Platform features require users to have Admin permissions, if users are logged in as a Viewer or Editor an information message will show. Added this info to the docs so users are aware that they need to be logged in as PMM Admin users.

Also updates `PMM_DBaaS_Kubernetes_Cluster_Panel.jpg` image.

### CODE REVIEW

**Review by PaulJacobs-percona — COMMENTED** [review]
What features exactly?
What does the user need to do to correctly use all Percona Platform features?

**Comment by tiagomotasantos** [comment]
> What features exactly?

Backup, DBaaS, Integrated Alerting and Security Checks.

> What does the user need to do to correctly use all Percona Platform features?

Be logged in as PMM Admin and enable the feature.

File: docs/using/platform/index.md [hunk 1]
```diff
@@ -6,3 +6,7 @@ The services comprise:
 
 - [Security Threat Tool](security-threat-tool.md)
 - [DBaaS](dbaas.md) 
+
+Currently some Percona Platform features are only available for PMM Admin users. If you are logged in as a user that has a Viewer or Editor role you'll see an 'insufficient access' message.
```
Comment (PaulJacobs-percona) [inline docs/using/platform/index.md:10]:
```suggestion
These Percona Platform features are currently only available for PMM Admin users:

- Backup;
- DBaaS;
- Integrated Alerting;
- Security Checks.
 
To use these features you must be logged in as a PMM Admin user and activate the features.
 
If you are logged in as a user that has a Viewer or Editor role you'll see an 'insufficient access' message when trying to use these features.
```

Comment (tiagomotasantos) [reply docs/using/platform/index.md]: Perfect @PaulJacobs-percona!

**Review by PaulJacobs-percona — APPROVED** [review]
Good job yourself @tiagomotasantos

**Pull request merged**